packages/react-app/src/models/token-amount.model.ts

```diff
@@ -2,5 +2,6 @@ import { TokenModel } from './token.model';
 
 export type TokenAmountModel = {
   parsedAmount: number;
+  parsedAmountStable?: number; // maybe create new type to handle it or use with PairModel
```

Comment:
thats seems fair for me, maybe renaming it  with usdValue, the parsed referred for the token amount parsed with decimals